spec/spec_helper.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
-require 'capybara/rspec'
-require 'selenium-webdriver'
 require 'byebug'
+require 'capybara/rspec'
+require 'io/console'
 require 'pry-byebug'
+require 'random_word'
```

Comment:
1. is this used in more than one place
2. where it is used ... does it need to be a random word?  

ok to make a ticket for this

Reply:
@ndushay yeah, it's used in 7 of our 9 feature specs. It's used to create random-looking text strings, such as titles and tags. It's more useful than hard-coding e.g. "Test Collection Title" because it gives us a string that we can look for later that isn't confused with another object.

Reply:
cool - thanks.